src/components/Bottom.tsx

```diff
@@ -1,30 +1,49 @@
 import React, { useState } from 'react';
-import { BottomNavigation, BottomNavigationAction, Box, Typography } from '@mui/material';
+import { BottomNavigation, BottomNavigationAction, Box, Typography, useTheme } from '@mui/material';
 import PlayCircleOutlineIcon from '@mui/icons-material/PlayCircleOutline';
 import StopCircleIcon from '@mui/icons-material/StopCircle';
 
-const Bottom: React.FC = () => {
-  const [coeRunning, setCoeRunning] = useState(false);
+const Bottom: React.FC<{ sidebarWidth: number; sidebarOpen: boolean }> = ({ sidebarWidth, sidebarOpen }) => {
+  const [maestroRunning, setMaestroRunning] = useState(false);
+  const theme = useTheme();
 
-  const toggleCoeState = () => setCoeRunning((prev) => !prev);
+  const toggleMaestroState = async () => {
+    try {
+      if (maestroRunning) {
+        await window?.electronAPI?.stopMaestro();
+      } else {
+        await window?.electronAPI?.startMaestro();
+      }
+      setMaestroRunning(!maestroRunning);
+    } catch (error) {
+      console.error('Error toggling Maestro:', error);
+    }
+  };
 
   return (
     <Box
       sx={{
-        width: 'calc(100% - 240px)',
+        width: '100%',
         position: 'fixed',
         bottom: 0,
-        left: '240px',
+        left: `${sidebarWidth}px`,
         bgcolor: 'background.paper',
-        boxShadow: 3,
+        boxShadow: theme.shadows[3],
+        transition: 'left 0.3s ease, width 0.3s ease',
```

Comment:
When left or width change, this will animate such change over 0.3 seconds with an easing function. It is mainly noticeable when the sidebar collapses, giving it a smooth animation instead of an instant change.
I will put the 0.3s as a constant

Reply:
TRANSITION_DURATION constant has been defined.